### PR TITLE
Don't strip triple backticks from internal API code blocks

### LINKED ISSUE #131: WP_CLI::runcommand() doc syntax highlighting
https://make.wordpress.org/cli/handbook/internal-api/wp-cli-runcommand/

The example code block is not parsed correctly, curly quotes, a single `-` instead of `--`

If you put some text (e.g. "Example:") in between the unordered list and the code block, the latter will be correctly highlighted

### PR DESCRIPTION
**Pull request by danielbachhuber**
WordPress.org knows how to process them just fine

Fixes #131

### CODE REVIEW

File: internal-api/wp-cli-run-command.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ To run a command using a new process with the same global parameters,
 use WP_CLI::launch_self(). To run a command using a new process with
```
Comment (Sidsector9) [inline internal-api/wp-cli-run-command.md]: @danielbachhuber We can put these functions in code blocks too

File: internal-api/wp-cli-run-command.md [hunk 2]
```diff
@@ -26,11 +26,11 @@ To run a command using a new process with the same global parameters,
...
 different global parameters, use WP_CLI::launch().
```
Comment (Sidsector9) [inline internal-api/wp-cli-run-command.md]: Same here

File: internal-api/wp-cli-utils-parse-ssh-url.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ Parse a SSH url for its host, port, and path.
 
 Similar to parse_url(), but adds support for defined SSH aliases.
```
Comment (Sidsector9) [inline internal-api/wp-cli-utils-parse-ssh-url.md]: Put parse_url() in codeblocks

**Comment by schlessera** [comment]
@Sidsector9 While you're absolutely right about putting these function/method calls into backticks, they are not directly relevant to the pull request at hand. IF @danielbachhuber would now fix these within the current pull request, it would muddy what the pull request is all about, i.e. what the title of the PR tells us.
In this case, it would be preferable to have a separate PR for these calls instead.

**Comment by Sidsector9** [comment]
@schlessera That makes sense! :+1:

**Pull request merged**